games/xworld3d/xworld3d.cpp

```diff
@@ -83,7 +83,7 @@ void X3Stadium::load_stadium(const std::string& item_path,
                                 true);
 }
 
-X3World::X3World(const std::string& conf, bool print_conf, bool big_screen) :
+X3World::X3World(const std::string& conf, bool print_conf, bool dry_run, bool big_screen) :
```

Comment:
No need to have dry_run as an argument. You can directly use FLAGS_x3_dry_run in this function.